src/cephadm/cephadm.py

```diff
@@ -8367,7 +8367,7 @@ def rm_repo(self) -> None:
 
     def install(self, ls: List[str]) -> None:
         logger.info('Installing packages %s...' % ls)
-        call_throws(self.ctx, ['apt-get', 'install', '-y'] + ls)
+        call_throws(self.ctx, ['apt-get', 'install', '-y', '--no-install-recommends'] + ls)
```

Comment:
Opened a patch for changing the recommends in the deb and it seemed to work from my testing in https://github.com/ceph/ceph/pull/52864#issuecomment-1671819894. If that's okay for you we can probably close this.

Reply:
That should do it it think. I finally realized why we were seeing it all of a sudden, I had added the cri-o repos from opensuse which probably also makes podman installable which is why it all of a sudden began installing with cephadm even when docker-ce was installed.  Now that docker-ce is in the recommended I assume it will solve that. I'll have to test my own stuff

Reply:
yeah, it shouldn't install anything extra id you have docker-ce now. Will point out the change is currently only merged into the main branch. We have a backport open that should allow us to get this into the next reef point release, but for now it's not so easy to test it unless you want to mess with dev builds.